website/docs/providers/rac/how-to-rac.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Create a Remote Access (RAC) provider
+---
+
+:::info
+This feature is in technical preview, so please report any Bugs you run into on [GitHub](https://github.com/goauthentik/authentik/issues)
+:::
+
+The RAC provider is a highly flexible feature for accessing remote machines. This document provides instructions for the basic creation and configuration of a RAC provider within a defined scenario.
+
+Fow more information about using a RAC provider, see the [Overview](./index.md) documentation. You can also view our [video on YouTube](https://www.youtube.com/watch?v=9wahIBRV6Ts) for setting up a RAC.
```

Comment:
Suggestion: "[our video on YouTube]".  It's a small change, but "our video" is one complete noun phrase and we should preserve the syntax of it.